challenges/functions.js

```diff
@@ -1,41 +1,53 @@
-// ==== Callbacks ====  
+// ==== Callbacks ====
 
 /* Step 1: Create a function that accepts a callback
   * Create a function named consume that can take 3 parameters.
   * The first two parameteres can accept any argument
   * The last parameter accepts a callback 
   * In the body of the function return the callback with the two parameters that you created
 */
-
+function consume(param1, param2, callback) {
+	callback(param1, param2);
```

Comment:
The `return` keyword is missing. The `return` is specified as part of the instructions, but why does this still work without the `return` keyword? What's the difference between using it or omitting it?